kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue

```diff
@@ -4,6 +4,7 @@
     :title="$tr('welcomeModalHeader')"
     :enableBgClickCancel="false"
     @cancel="emitCloseModal"
+    @keyup.enter="emitCloseModal"
```

Comment:
The `core-modal` plugin has a [built-in way](https://github.com/learningequality/kolibri/pull/2637/commits/0d3c8443737e3b43074a554ced22cd21991d2d24) of supporting this behavior.

Reply:
Oh I see. This fix is much simpler. Thanks Devon!